docs/content/Getting-Started/Getting-Started-Docker-Compose.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Docker Compose
+permalink: /getting-started/docker/compose
+category: Getting Started
+menuOrder: 3
+---
+
+This guide will help you get Cube.js running using Docker Compose.
+
+> LOOM VIDEO HERE
+
+## 1. Create a Docker Compose file
+
+Create a `docker-compose.yml` file with the following content:
+
+```yaml
+version: '2.2'
+
+services:
+  cube:
+    image: cubejs/cube:latest
+    ports:
+      # 4000 is a port for Cube.js API
```

Comment:
I feel the description for ports may be improved, not very clear.

Reply:
I pulled that straight from our template; if we update it here we should probably also do the same in the templates 

/cc @vasilev-alex